editoast/src/views/infra/auto_fixes/mod.rs

```diff
@@ -233,9 +234,8 @@ fn reduce_operation(
             } = update;
             debug_assert_eq!(railjson_object.get_id(), &obj_id);
             debug_assert_eq!(railjson_object.get_type(), obj_type);
-            railjson_object
-                .patch(&railjson_patch)
-                .map(|railjson_object: RailjsonObject| Operation::Create(Box::new(railjson_object)))
+            patch_infra_object(&railjson_object, &railjson_patch)
```

Comment:
This one will need to modify the initialization in the destructuring above.

```suggestion
            patch_infra_object(&infra_object, &railjson_patch)
```